src/features/CourtCaseList/tags/ResolvedTag.tsx

```diff
@@ -0,0 +1,21 @@
+import If from "components/If"
+import { Tag } from "govuk-react"
+import { textGrey } from "utils/colours"
+import { useCustomStyles } from "../../../../styles/customStyles"
+
+const ResolvedTag: React.FC<{ isResolved: boolean }> = (props: { isResolved: boolean }) => {
+  const classes = useCustomStyles()
+  return (
+    <If condition={props.isResolved}>
+      <Tag
```

Comment:
I was speaking to Helena and Jess recently and it turns out that the MoJ have their own component for this that we can use, see https://design-patterns.service.justice.gov.uk/components/badge/

I think we can just set the class of `moj-badge--grey` on this and not have to use a custom style, I'm going to create a ticket to update the rest